cpu/cc26xx_cc13xx/Kconfig

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2020 Locha Inc
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config CPU_CC26XX_CC13XX
```

Comment:
I understand that the `cc26xx_cc13xx` is just common code, not used directly by any board. I think in that case we can rename this symbol to `CPU_COMMON_CC26XX_CC13XX` to indicate that.

Reply:
Makes sense, I've rename it to `CPU_COMMON_CC26XX_CC13XX`